taskcluster/tc-py-utils.sh

```diff
@@ -72,14 +72,14 @@ maybe_setup_virtualenv_cross_arm()
   fi;
 
   if [ "${OS}" != "Linux" ]; then
-    echo "Only for Linux";
-    exit 1;
+    echo "Only for Linux/ARM arch";
+    exit 0;
```

Comment:
These checks are simply skipping the entire test: https://community-tc.services.mozilla.com/tasks/S_rvCFKDRqOzJQW9HddmfA/runs/0/logs/https%3A%2F%2Fcommunity-tc.services.mozilla.com%2Fapi%2Fqueue%2Fv1%2Ftask%2FS_rvCFKDRqOzJQW9HddmfA%2Fruns%2F0%2Fartifacts%2Fpublic%2Flogs%2Flive.log#L2964

Reply:
I think these need to be return instead of exit? My Bash skills are failing me.